### DISCUSSION TITLE
Score calculator for version 7?

### DISCUSSION THREAD

**Discussion by verlok** [GoogleChrome/lighthouse#11865] [Q&A] [answered]
**Summary**

Hey there,
we're looking at https://googlechrome.github.io/lighthouse/scorecalc/ to see how the score is calculated by LightHouse.

But the latest version of the calculation is v6. 
Will v7 change that?

Thanks

**Comment by patrickhulce** [comment]
Nope, v7 had no Performance score weight changes from v6.

**Comment by paulirish** [comment]
It's true that the "see calculator" link now breaks the view: 

https://googlechrome.github.io/lighthouse/scorecalc/#FCP=671&SI=671&LCP=762&TTI=671&TBT=0&CLS=0&FCI=671&FMP=671&device=mobile&version=7.0.0

i'll fix that.

**Comment by paulirish** [comment, marked as answer]
Fixed! https://github.com/paulirish/lh-scorecalc/pull/31

and deployed. https://googlechrome.github.io/lighthouse/scorecalc/
